form.js

```diff
@@ -0,0 +1,85 @@
+const Form = document.getElementById('booksform');
+const Bookscontainer = document.querySelector('.books');
+const bookname = Form.title;
+const writer = Form.name;
+
+let title = ''; let
+  namey = '';
+
+const books = [];
+
+function Book(title, name, node = null) {
+  this.id = books.length;
+  this.title = title;
+  this.name = name;
+  this.node = node;
+}
+
+const updateStorage = () => {
+  let string = '';
+  books.forEach((book) => {
+    string = `${string}${JSON.stringify(book)}|`;
+  });
+  string = string.substring(0, string.length - 1);
+  localStorage.setItem('books', string);
+};
+
+const createBooksElements = (book) => {
+  const Booksdiv = document.createElement('div');
+  const BookTitle = document.createElement('p');
+  const BookWritter = document.createElement('p');
+  const remove = document.createElement('button');
+  remove.innerHTML = 'Remove';
+  remove.addEventListener('click', () => {
```

Comment:
- [ ] I think it's better to fix this remove button functionality. It's not working after deleting and recreating books. Consider using the [filter](https://www.w3schools.com/jsref/jsref_filter.asp).